apis/Google.Cloud.Diagnostics.AspNet/docs/index.md

```diff
@@ -42,6 +42,10 @@ installed, run the following command in a Google Cloud SDK Shell:
 
 [!code-cs[](obj/snippets/Google.Cloud.Diagnostics.AspNet.AspNet.txt#InitializeTrace)]
 
+## Trace Outgoing Http Requests
```

Comment:
.NET docs use all uppercase "HTTP". We may want to be consistent with that (same comment for all lowercase "http" uses in /// docs).